tests/irepela/homework_9/test_fraction.py

```diff
@@ -0,0 +1,102 @@
+import pytest
+from irepela.homework_9.fraction import Fraction
+
+
+class Custom:
+
+    def __new__(cls):
+        return 5
```

Comment:
it does not make sense, you do not create object of Custom type
you create object of int type

you need object with __radd__ method